pdoc/__init__.py

```diff
@@ -1049,7 +1049,7 @@ def __init__(self, name, module, func_obj, cls=None, method=False):
         """
         super(Function, self).__init__(name, module, inspect.getdoc(func_obj))
 
-        self.func = func_obj
+        self.func = inspect.unwrap(func_obj)
```

Comment:
According to the docs for `inspect.unwrap`, it is new in Python 3.4. `pdoc` *must* remain compatible with at least Python 2.7, so you'll need to make this backwards compatible before I can merge it.